functions/python-sqs-trigger-hello-world/main.tf

```diff
@@ -41,61 +41,48 @@ resource "scaleway_function" "main" {
 
 # SQS trigger resources
 
-resource "scaleway_mnq_namespace" "main" {
-  name     = "serverless-examples"
-  protocol = "sqs_sns"
-}
-
-resource "scaleway_mnq_credential" "main" {
-  name         = "serverless-examples"
-  namespace_id = scaleway_mnq_namespace.main.id
-  sqs_sns_credentials {
-    permissions {
-      can_publish = true
-      can_receive = true
-      can_manage  = true
-    }
+resource "scaleway_mnq_sqs_credentials" "main" {
+  permissions {
+    can_publish = true
+    can_receive = true
+    can_manage  = true
```

Comment:
Leaving a nitpick/comment so that you know I've read the MR :sunglasses: 

Do we even need `can_manage` in those examples?